docs/source/reference/pragmas.rst

```diff
@@ -14,17 +14,127 @@ small niche of pragmas apply directly to one or more arguments instead of the co
     This page is a work in progress. If you know about a pragma that is not described yet, please consider
     submitting a pull request!
 
+Global pragmas
+====================
+
+``%language``
+--------------------
+
+Enable language extensions.  Currently, the only extension is ``ElabReflection``.
+
+.. code-block:: idris
+
+   %language ElabReflection
+
+``%default``
+--------------------
+
+Set the default totality requirement for functions.  The initial value is `covering`, which
```

Comment:
Maybe, it's worth adding that `total` can be an initial value if an appropriate flag is passed to the compiler